tests/platform_tests/test_platform_info.py

```diff
@@ -173,7 +173,9 @@ def get_healthy_psu_num(duthost):
     """
     PSUUTIL_CMD = "sudo psuutil status"
     healthy_psus = 0
-    psuutil_status_output = duthost.command(PSUUTIL_CMD)
+    psuutil_status_output = duthost.command(PSUUTIL_CMD, module_ignore_errors=True)
+    if psuutil_status_output['rc'] != 0:
```

Comment:
@yutongzhang-microsoft if rc !=0, is there any possibility that the command doesn't work or is something wrong on physical testbed? If so, how to handle this scenario?

Reply:
I will add another condition here, `if duthost.facts["asic_type"] == "vs"`